### PR TITLE
added deploy for GA

### PR DESCRIPTION
**Pull request by watchpartyappmain**

### CODE REVIEW

File: .github/workflows/cli-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Run client install and test
+        run: |
+          cd WatchParty/client/
```
Comment (PredatorFeesh) [inline .github/workflows/cli-deploy.yml:26]: It's no longer WatchParty/client/ , we are pushing the frontend branch directly into here. No need for any `cd`

File: .github/workflows/cli-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      matrix:
+        node-version: [13.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Run client install and test
```
Comment (PredatorFeesh) [inline .github/workflows/cli-deploy.yml:24]: Also, the name here should be changed to "Run frontend tests"

File: .github/workflows/cli-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        run: |
+          cd WatchParty/client/
+          npm install
+          npm run test
+        env:
+          CI: true
+      - name: Deploy to Heroku
+        env:
+          HEROKU_API_TOKEN: ${{ secrets.HEROKU_API_TOKEN }}
```
Comment (PredatorFeesh) [inline .github/workflows/cli-deploy.yml:33]: Did you set this in our secrets? Go into GitHub `Settings -> Secrets`

File: .github/workflows/cli-deploy.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          HEROKU_APP_NAME: "watch-party-test-web-client"
+        if: github.ref == 'refs/heads/master' && job.status == 'success'
```
Comment (PredatorFeesh) [inline .github/workflows/cli-deploy.yml:35]: Also, take this out. We don't need this yet (we will later), for now it'll just keep this from every running.

File: .github/workflows/cli-deploy.yml (hunk 3 above)
Comment (watchpartyappmain) [reply .github/workflows/cli-deploy.yml:33]: Got it set up first step yep

Comment (PredatorFeesh) [reply .github/workflows/cli-deploy.yml:33]: nice 😄

File: .github/workflows/cli-deploy.yml (hunk 4 above)
Comment (watchpartyappmain) [reply .github/workflows/cli-deploy.yml]: removed if

**Pull request merged**